autoarena/service/model.py

```diff
@@ -96,13 +96,10 @@ def upload_responses(project_slug: str, model_name: str, df_response: pd.DataFra
             raise BadRequestError(str(e))
         if len(df_response) == 0:
             raise BadRequestError("Responses must not be empty")
+        df_response = df_response.copy().replace({np.nan: ""})
```

Comment:
If we ever make use of columns beyond  `prompt` or `response`, we may want to restrict it to `df_response[["prompt", "response"]]`.